requirements-test-libraries.txt

```diff
@@ -222,6 +222,7 @@ yapf==0.40.2
 xmlschema==2.5.1
 pysaml2==7.5.0; python_version >= '3.9'
 pysaml2==7.3.0; python_version < '3.9'
+toga==0.4.6; python_version >= '3.8'

```

Comment:
The python version restrictions should apply to what we currently use on our CI, so `; python_version >= '3.8'` is redundant.